indico/modules/events/registration/controllers/management/regforms.py

```diff
@@ -378,11 +378,13 @@ class RHRegistrationFormModify(RHManageRegFormBase):
     def _process(self):
         min_data_retention = data_retention_settings.get('minimum_data_retention')
         max_data_retention = data_retention_settings.get('maximum_data_retention') or timedelta(days=3650)
+        regform_retention_days = self.regform.retention_period.days // 7 if self.regform.retention_period else None
```

Comment:
hello @ThiefMaster, didn't see this before, the unit of this field is weeks, not days. So should be `regform_retention_weeks`